client/src/components/uis/__tests__/MessageListItem.test.tsx

```diff
@@ -53,25 +55,32 @@ const QueryWrapper: React.FC<QueryWrapperProps> = (props) => {
           id: 'previous-message-id',
         }}
         nextItemDate="2021-03-19T00:57:16.762Z"
+        userId={props.userId}
       />
     </Suspense>
   );
 };
 
-const mockEnvironment = createMockEnvironment();
+let mockEnvironment;
 
-mockEnvironment.mock.queueOperationResolver((operation) =>
-  MockPayloadGenerator.generate(operation, {
-    Message(_, generateId) {
-      return {
-        ...SAMPLE_MESSAGE,
-        id: `test-message-${generateId()}`,
-      };
-    },
-  }),
-);
+// beforeEach(() => ();
```

Comment:
I think we can remove this :)

Reply:
All above suggestion was applied. thans :)